### DISCUSSION TITLE
multiple `create_account_route`s: is this possible?

### DISCUSSION THREAD

**Discussion by jf** [jeremyevans/rodauth#415] [General]
I am looking at creating an application where there are different types of users... and I'd like to have a different route to handle the signup process for each of them. Is this possible? I understand from the docs that `create_account_route` only works with 1 route; what can I do if I want Rodauth to be able to create accounts from multiple different routes?

**Comment by jeremyevans** [comment]
You can have multiple Rodauth configurations in the same app: https://rodauth.jeremyevans.net/rdoc/files/README_rdoc.html#label-With+Multiple+Configurations

That's what you want to do if you have multiple create_account routes in one application.
